SeaPublicWebsite/Controllers/CookieController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using GovUkDesignSystem.GovUkDesignSystemComponents;
+using Microsoft.AspNetCore.Mvc;
+using SeaPublicWebsite.Models.Cookies;
+using SeaPublicWebsite.Services.Cookies;
+
+namespace SeaPublicWebsite.Controllers;
+
+public class CookieController: Controller
+{
+    private readonly CookieService cookieService;
+
+    public CookieController(CookieService cookieService)
+    {
+        this.cookieService = cookieService;
+    }
+
+    [HttpGet("/cookies")]
```

Comment:
The `/` at the beginning here means that we may look like we're setting cookies for all of GOV.UK